### PR TITLE
(Enhancement): Add icons for payment methods

### PR DESCRIPTION
**Pull request by AlicanAkyuz**
The type of this PR is: **Enhancement**

### Description

This PR adds icons for payment methods of bank transfer and credit card in the checkout flow. 

#### Photos
Screenshot 2022-05-17 at 16 45 51

### CODE REVIEW

**Review by fladson — APPROVED** [review]

**Review by jacobherrington — DISMISSED** [review]
👍

File: src/v2/Apps/Order/Routes/Payment/PaymentContent.tsx [hunk 1]
```diff
@@ -74,9 +76,22 @@ export const PaymentContent: FC<Props> = props => {
         >
           <BorderedRadio
             value="bank_debit"
-            label="Bank transfer (US bank account)"
+            label={
+              <>
+                <InstitutionIcon fill="green100" />
+                <Text ml={1}>Bank transfer (US bank account)</Text>
```
Comment (dzucconi) [inline src/v2/Apps/Order/Routes/Payment/PaymentContent.tsx:82]: I realize this isn't a required prop... but for the moment: Text should always specify a variant

Comment (AlicanAkyuz) [reply src/v2/Apps/Order/Routes/Payment/PaymentContent.tsx]: thanks for flagging it, I've just seen the ticket on WP board 👍

File: src/v2/Apps/Order/Routes/Payment/PaymentContent.tsx [hunk 2]
```diff
@@ -74,9 +76,26 @@ export const PaymentContent: FC<Props> = props => {
         >
           <BorderedRadio
             value="bank_debit"
-            label="Bank transfer (US bank account)"
+            label={
+              <>
+                <InstitutionIcon fill="green100" />
```
Comment (rquartararo) [inline src/v2/Apps/Order/Routes/Payment/PaymentContent.tsx:81]: I think both the icons should be `black60` according to the figma.

**Review by rquartararo — APPROVED** [review]
Looks great, just the icon colors that need to be updated!

**Pull request merged**